Azeem/src/DLMFtex2Wiki.py

```diff
@@ -1,795 +1,794 @@
-#Version 13: Minor Fixes
-#Convert tex to wikiText
-import csv #imported for using csv format
-import sys #imported for getting args
-import os #imported for copying file
-import tex2Wiki as KLS
- #Change Accordingly based on project directory.
-def is_number(char):
-    try:
-        "".join(["0"+str(float(char)) if float(char) < 10 else float(char)])
-        return True
-    except ValueError:
-        return False
+# Version 15: Minor Fixes
+# Convert tex to wikiText
+import csv  # imported for using csv format
+import sys  # imported for getting args
+from shutil import copyfile
+from tex2Wiki import append_text, append_revision, getString,\
+    getSym, getEq, secLabel, getEqP, unmodLabel, isnumber
+
+
+
+
 
 def modLabel(label):
-    #label.replace("Formula:KLS:","KLS;")
-    isNumer=False
-    newlabel=""
-    num=""
-    for i in range(0,len(label)):
-        if isNumer and not is_number(label[i]):
-            if len(num)>1:
-                newlabel+=num
-                isNumer=False
-                num=""
+    isNumer = False
+    newlabel = ""
+    num = ""
+    for i in range(0, len(label)):
+        if isNumer and not isnumber(label[i]):
+            if len(num) > 1:
+                newlabel += num
+                num = ""
             else:
-                newlabel+="0"+str(num)
-                num=""
-                isNumer=False
-        if is_number(label[i]):
-            isNumer=True
-            num+=str(label[i])
+                newlabel += "0" + str(num)
+                num = ""
+        if isnumber(label[i]):
+            isNumer = True
+            num += str(label[i])
         else:
-            isNumer=False
-            newlabel+=label[i]
-    if len(num)>1:
-        newlabel+=num
-    elif len(num)==1:
-        newlabel+="0"+num
-    return(newlabel)
-
-    
-def DLMF(n):
-    for iterations in range(0,1):
-        try:
-            tex=open(sys.argv[1],'r')
-            wiki = open(sys.argv[2], 'w')
-            glossary = open(sys.argv[3],"r")
-            main=open(sys.argv[4],"r")
-            lLinks = open(sys.argv[5], 'r')
-        except (IOError,IndexError):
-            tex = open("../../data/ZE.3.tex","r")
-            wiki = open("../../data/ZE.4.xml","w")
```

Comment:
@pwang00 the red text on the left shows the code that was deleted. The green text on the right, shows was added instead.